CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Ewon Flexy Extensions Library Changelog
 
+## Version 1.6.0
```

Comment:
I assume this is the point at which you would tag the repo for a release. I dont think its documented but we have been setting the commit title to "RELEASE vx.y.z" to consistently mark this. 


@alexjhawk  should we add this to CONTRIBUTING.md?

Reply:
@TomKimsey  Yes, this should be added. Incrementing the version and any associated change logs needs to be in the last commit. Here is an example: https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/commit/294353ed050dfb7bc6dc395d4faf91a1ed619555.

The formatting in CHANGELOG.md is especially important, as it could cause the build and release actions to fail if not followed.

Reply:
ok.. will change commit title.

Reply:
Commit title should be "RELEASE: v1.6.0"

Reply:
+1